docs/deployment/vanilla/README.md

```diff
@@ -1,6 +1,8 @@
 # Deploying Kubeflow on EKS
 
-This guide describes how to deploy Kubeflow on AWS EKS
+This guide describes how to deploy Kubeflow on AWS EKS. This vanilla version has very minimal changes to the upstream Kubeflow manifests. Here are the changes  
```

Comment:
how to deploy Vanilla Kubeflow on EKS?

same for the heading of this readme Deploying Vanilla Kubeflow on EKS?

wdyt

Reply:
changed to "how to deploy Vanilla Kubeflow on EKS"
but kept the heading